perspective_reddit_bot/log_subreddit_comments.py

```diff
@@ -50,10 +50,11 @@ def comment_url(comment):
   return 'https://reddit.com' + comment.permalink
 
 
-def append_record(output_path, record):
+def append_records(output_path, records):
```

Comment:
Just curious, why change this if it's only being used for one record at a time at the moment?

Reply:
it's used in the next PR